.evergreen.yml

```diff
@@ -75,6 +75,7 @@ functions:
       args: ["${make_args|}", "${target}"]
       background: true
       env:
+        SETTINGS_OVERRIDE: file
```

Comment:
this is necessary to use the evg cloud mockManager during spawn volume e2e tests